sdk/generic-sdk-core/generic-core-http-okhttp/pom.xml

```diff
@@ -70,7 +72,19 @@
     <dependency>
       <groupId>com.squareup.okhttp3</groupId>
       <artifactId>okhttp</artifactId>
-      <version>4.10.0</version> <!-- {x-version-update;com.squareup.okhttp3:okhttp;external_dependency} -->
+      <version>4.12.0</version> <!-- {x-version-update;com.squareup.okhttp3:okhttp;external_dependency} -->
+    </dependency>
+
+    <dependency>
```

Comment:
what are these new dep for?

Reply:
Is this not needed in azure-core-http-okhttp then?

Reply:
azure-core-http-okhttp doesn't have the additional implementation generic-core-http-okhttp has, at least not yet.